packages/cloud-functions/src/cico/simplexEventPolling.schema.sql

```diff
@@ -0,0 +1,14 @@
+event_id: STRING,
+name: STRING,
+payment_id: STRING,
+payment_status: STRING,
+payment_created_at: STRING,
+payment_partner_id: FLOAT,
+payment_updated_at: STRING,
+payment_crypto_currency: STRING,
+payment_fiat_total_amount_amount: FLOAT,
+payment_fiat_total_amount_currency: STRING,
+payment_crypto_total_amount_amount: FLOAT,
+payment_crypto_total_amount_currency: STRING,
+payment_partner_end_user_id: STRING,
+timestamp: TIMESTAMP
```

Comment:
What part is using this file?

Reply:
This isn't being used directly by the code. It's the schema for the BigQuery table that we are creating for Simplex events.